WordPress/wordpress-playground
# Add more info to crash reports

Fixes #1152

## What is this PR doing?

It adds environment context data and the blueprint to crash reports. 

## What problem is it solving?

This should help us debug crash reports.

## How is the problem addressed?

By adding more data to the crash report. 

## Testing Instructions

- Add debug code (see [code removed in this commit ](https://github.com/WordPress/wordpress-playground/pull/1253/commits/653735a51dd07bfe6251e26300250cf775c2d1c3))
- Open Playground locally
- When the error report modal opens, add a description and click submit
- In the browse console see the context data
